.github/workflows/testing.yml

```diff
@@ -204,9 +204,9 @@ jobs:
           if [[ ${{needs.check_files.outputs.run_job}} == false ]]; then
             exit 0
           fi
-          docker create --user dev --name taichi_test ghcr.io/taichi-dev/taichidev-cpu-ubuntu18.04:v0.1.0 /home/dev/unix_docker_test.sh $PY $GPU_TEST $TI_WANTED_ARCHS
+          docker create -v `pwd`/shared/build:/home/dev/build --user dev --name taichi_test ghcr.io/taichi-dev/taichidev-cpu-ubuntu18.04:v0.1.0 /home/dev/unix_docker_test.sh $PY $GPU_TEST $TI_WANTED_ARCHS
           docker cp .github/workflows/scripts/unix_docker_test.sh taichi_test:/home/dev/unix_docker_test.sh
-          docker cp wheel/*.whl taichi_test:/home/dev/
+          docker cp shared/*.whl taichi_test:/home/dev/
```

Comment:
hmmm do we only copy the whls over? We probably also need the build/ dir?

Reply:
We already mounted the build dir into the container:
```
docker create -v `pwd`/shared/build:/home/dev/build ...
```